ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -0,0 +1,22 @@
+---
+# This task will install the standard actions and packages available in openwhisk-catalog.
+
+- name: the openwhisk-catalog URL and the location to download
+  set_fact:
+    openwhisk_catalog_url: "https://github.com/openwhisk/openwhisk-catalog.git"
+    openwhisk_catalog_home: "{{ openwhisk_home }}/../openwhisk-catalog"
```

Comment:
Should not hardcore where to install the catalog. How about an variable with this as a default.